registry/config.go

```diff
@@ -0,0 +1,56 @@
+package registry
```

Comment:
I think this file belongs in `config/`. It does return registry types, but it's operating on the config file more than anything else.